site/content/en/docs/overview/_index.md

```diff
@@ -15,6 +15,15 @@ You can install Kueue on top of a vanilla Kubernetes cluster. Kueue does not rep
 * Compute resources are elastic and can be scaled up and down.
 * Compute resources are heterogeneous (in architecture, availability, price, etc.).
 
+<figure>
+  <img src="/images/theory-of-operation.svg" alt="theory-of-operation" style="width:100%">
+  <figcaption>High-level Kueue operation</figcaption>
```

Comment:
Make it a subtitle

Reply:
Do you mean add the text above? As a H3/H4?

Reply:
yes